### DISCUSSION TITLE
Get components of feature in Bootstrapper application

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#8813] [Questions]
I can get feature information in the Bootstrapper application using "WixPackageFeatureInfo" which is present in BootstrapperApplicationData.xml.

Components information is not available in BootstrapperApplicationData.xml. How do I access components of features?

**Comment by robmen** [comment]
Why would you want to do that?

**Comment by kalyani1217** [comment]
@robmen,  I am delivering multiple configuration files. If the customer has modified any of these files, I should not install them during the update setup. However, if the customer has not made any changes, I need to override the configuration files during the installation.

If I can access the component information, I can check whether each configuration file has been modified. Based on this, I will set the component state to either "absent" or "present."

**Reply by robmen** [reply]
I saw your response. @'ing me doesn't make me want to help you. With all due respect, I don't hang out on this community list to solve your installation design issues. If you want to make demands of my time, [become a customer](https://www.firegiant.com/services/support-programs/).

Note: I asked my question to understand the scenario and see if there was a use case that needed to be supported.
